product/charts/layouts/daily_perf_charts/AvailabilityZone.yaml

```diff
@@ -26,6 +26,27 @@
     :columns: 
     - derived_vm_count_on
 
+- :title: Memory (MB)
+  :type: Line
+  :columns:
+  - derived_memory_used
+  - min_derived_memory_used
+  - max_derived_memory_used
+  - trend_max_derived_memory_used
+  - resource.derived_memory_used_high_over_time_period
+  - resource.derived_memory_used_low_over_time_period
+  :menu:
+  - Chart-Current-Hourly:Hourly for this day
+  - Chart-VMs-topday:Top Instances on this day
+#  - Timeline-Current-Daily:Daily events in this Availability Zone
+#  - Timeline-Current-Hourly:Hourly events in this Availability Zone
```

Comment:
It's that way *everywhere else* -- so I kept it consistent. If this should go away, someone should make the change consistently in a separate PR.